frontend/src/Pages/SentenceDetails.tsx

```diff
@@ -176,22 +186,6 @@ const SentencesDetails = () => {
           setConnectivityStatements(
             sentence.connectivity_statements.sort((a, b) => a.id - b.id),
           );
-          if (
```

Comment:
I took the decision of removing the initial alert from the statement because in that page the user can still try to edit the statement (even if they are not the owner) which would trigger the appearance of the ownership alert. That doesn't seem to be the case for the sentences. In other words, with this change, I don't believe it's possible to re-take the ownership of a sentence. I propose that we don't remove this initial prompt from the sentences.

Reply:
discussed with @Salam-Dalloul